### PR TITLE
Create specific script directory acceptante tests

### PR DESCRIPTION
**Pull request by flopezag**
### REVIEWERS

@chemaper @jframos 
### DESCRIPTION
- Move unit test resources files under unit test directory.
- Separate glancesync acceptance tests from script acceptance tests.
- Redefine the Jenkins jobs related to Build and Acceptance.
- Allow the execution of Acceptance tests from different branchs.
- Change README content accordingly tho the previous changes.
- Check style
### TEST

Locally in IDE and CLI, Jenkins and Travis

### CODE REVIEW

**Comment by chemaper** [comment]
The title of the pull request and the description does not match, because the PR is composed of several unrelated changes. It may be OK when each change only affects a few files, but here there are a lot and it is difficult to track all the changes. I think it would have been better to make independent pull requests.

**Comment by chemaper** [comment]
LGTM

File: tests/acceptance/commons/__init__.py [hunk 1]
```diff
@@ -20,3 +20,5 @@
 #
 # For those usages not covered by the Apache version 2.0 License please
 # contact with opensource@tid.es
+
+__author__ = "fla"
```
Comment (jframos) [inline tests/acceptance/commons/__init__.py]: New line at the end of the file.

File: tests/acceptance/features/scripts/environment.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    context.config = load_project_properties()
+    context.master_region_name = \
+        context.config[PROPERTIES_CONFIG_GLANCESYNC][PROPERTIES_CONFIG_GLANCESYNC_MASTER_REGION_NAME]
+
+
+def before_scenario(context, scenario):
+    """
+    HOOK: To be executed before each Scenario:
+        - Init test variables
+        - Backup GlanceSync configuration file
```
Comment (jframos) [inline tests/acceptance/features/scripts/environment.py]: I think comments of these hooks are the result of Copy&Paste

File: tests/acceptance/commons/__init__.py (hunk 1 above)
Comment (flopezag) [inline tests/acceptance/commons/__init__.py]: Done

**Comment by jframos** [comment]
LGTM

**Pull request merged**